panther_manager/include/panther_manager/plugins/action/call_set_led_animation_service_node.hpp

```diff
@@ -8,7 +8,7 @@
 namespace panther_manager
 {
 
-class CallSetLedAnimationService : public panther_manager::RosServiceNode<panther_msgs::SetLEDAnimation>
+class CallSetLedAnimationService : public RosServiceNode<panther_msgs::SetLEDAnimation>
 {
 public:
   CallSetLedAnimationService(const std::string & name, const BT::NodeConfig & conf);
```

Comment:
As in `set_bool` everything is public, `nh_` is redefined, `srv_name_` as well. This time, provided ports is overwriting one from `RosServiceNode`, so `timeout` cannot be set. We will have to see if this class can be header only